### ISSUE TITLE
Question about using Pytorch-LiteFlownet during retraining/fine-tuning

### ISSUE THREAD

**Issue by jayrmh** [hzwer/arXiv2021-RIFE#248]
Hey guys! I love your work, and I am working on a survey paper exploring interpolation algorithms in medical domain. So far your method is one of the best performing!

I would to fine-tune the RIFE HDv3 model on my dataset which has image triplets. I have also gotten Ptroch-LiteFlownet to create optical flow for my dataset triplets. My question is how do I incorporate these flows into the train.py file, and which block of code do I need to modify to achieve this task.

Thank you, and I hope to hear from you soon.

**Comment by hzwer** [comment]
Hi, in our current version, we do not need to generate flow label for training. If you need this supervise, I advise add flownet to model.py add modify the loss_cons.
